storage/src/test/java/org/apache/uniffle/storage/handler/impl/LocalFileHandlerTestBase.java

```diff
@@ -44,16 +44,13 @@
 public class LocalFileHandlerTestBase {
   private static AtomicLong ATOMIC_LONG = new AtomicLong(0L);
 
-  public static void reset() {
-    ATOMIC_LONG = new AtomicLong(0L);
-  }
-
-  public static void writeTestData(
+  public static List<Long> writeTestData(
       ShuffleWriteHandler writeHandler,
       int num, int length,
       Map<Long, byte[]> expectedData,
       Set<Long> expectedBlockIds) throws Exception {
     List<ShufflePartitionedBlock> blocks = Lists.newArrayList();
+    List<Long> blockIds = Lists.newArrayList();
```

Comment:
Why not use `Set<Long> expectedBlockIds` directly?

Reply:
Can you give an example how to use `expectedBlockIds` in `calcSegmentBytes`?
I think we should preserve order and exclude the last block.

Reply:
Like this? 
![image](https://user-images.githubusercontent.com/8609142/212585816-27817581-87fb-4ae2-af18-b46fcc71ccc7.png)

Reply:
Thanks, I got it when you mean `TreeSet`. However, I'm wondering if we should keep the current `writeTestData()` signature.